docs/reference/data-streams/downsampling-ilm.asciidoc

```diff
@@ -0,0 +1,548 @@
+                                                            "type": "float"
+                                                        }
+                                                    }
+                                                },
+                                                "node": {
+                                                    "properties": {
+                                                        "pct": {
+                                                            "type": "float"
+                                                        }
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "workingset": {
+                                            "properties": {
+                                                "bytes": {
+                                                    "type": "long",
+                                                    "time_series_metric": "gauge"
+                                                }
+                                            }
+                                        }
+                                    }
+                                },
+                                "name": {
+                                    "type": "keyword"
+                                },
+                                "start_time": {
+                                    "type": "date"
+                                }
+                            }
+                        },
+                        "host": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        },
+                        "namespace": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        },
+                        "node": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        },
+                        "pod": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+----
+// TEST[continued]
+
+////
+[source,console]
+----
+DELETE _index_template/*
+----
+// TEST[continued]
+////
+
+[discrete]
+[[downsampling-ilm-ingest-data]]
+==== Ingest time series data
+
+Use a bulk API request automatically create your TSDS and index a set of ten
+documents.
+
+**Important:** Before running this bulk request you need to update the
+timestamps to within three to five hours after your current time. That is,
+search `2022-06-21T15` and replace with your present date, and adjust the hour
+to your current time plus three hours.
```

Comment:
Will users have to adjust timestamps? I am afraid this example becomes overly complicated.

Reply:
Yeah, I wish a knew a better approach for this. We have the same problem in the "[Create a TSDS](https://www.elastic.co/guide/en/elasticsearch/reference/master/set-up-tsds.html#create-tsds)" example, where we say `To test the following example, update the timestamps to within the 24 hours after your current time.`

Do you have any ideas? I suppose we could run a script to generate the json request, but that also seems kind of complicated. If you think the need to adjust timestamps makes the example not worth including, I can remove it.

Reply:
Unfortunately, I don't have anything better. If it works for the "Create TSDS" example, let's keep it here too.